### DISCUSSION TITLE
Is there a solution for it? - ERROR  (node:20253)

### DISCUSSION THREAD

**Discussion by ghost** [nuxt/nuxt#16401] [Questions]
npx nuxi init (project)
Nuxi 3.0.0-rc.13
 ERROR  (node:20253) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)

**Comment by TheAlexLichter** [comment]
This is just a warning and should not cause any problems during development ☺️

**Reply by ghost** [reply]
thx :)

**Reply by TheAlexLichter** [reply]
No problem! 

PS: Feel free to mark this reply as "answer" so others know the discussion is resolved ☺️
